include/evm_runtime/evm_contract.hpp

```diff
@@ -23,20 +24,8 @@ class [[eosio::contract]] evm_contract : public contract
 {
 public:
    using contract::contract;
+   evm_contract(eosio::name receiver, eosio::name code, const datastream<const char*>& ds) : contract(receiver, code, ds), _config(get_self()) {}
 
-   struct fee_parameters
```

Comment:
`fee_parameters` should continue to be defined here (or alternatively another header file if preferred) since it is needed by the `setfeeparams` action.

Then the definition can be brought into the `config_wrapper.cpp` file to implement the `set_fee_parameters` member function in `config_wrapper`. The declaration of the member function in `config_wrapper.hpp` can simply forward declare `fee_parameters` without defining it.